---
# -*- Mode: HTML; tab-width: 4; indent-tabs-mode: nil; -*-
# vim:set ft=html ts=4 sw=4 sts=4 autoindent:

layout: base
title:  "Annotation examples"
cmeta:  "Examples and case-studies of annotations produced or visualised using
    brat."
---

<h1>examples</h1>

<p>
    brat can be applied in many ways and is used in several
    completed and ongoing annotation efforts and other tasks:
</p>

<ul>
    <li><a href="#annotation-examples">annotation examples in brat</a></li>
    <li><a href="#corpus-examples-brat">corpora annotated using brat</a></li>
    <li><a href="#other-uses">other uses of brat</a></li>
</ul>

<!-- ############################################################ -->
      
<h1 id="annotation-examples">annotation examples in brat</h1>

<p>
    A variety of annotation tasks that can be performed in brat
    are introduced below using examples from available corpora.
	The examples discussed in this section have been originally
	created in various tools other than brat and converted into
	brat format. Converters for many of the original formats are
	distributed with brat.
	In the selection of examples included here, priority has been
	given to tasks with freely available data.
    <!-- 	(For example, although -->
    <!-- 	any CoNLL shared task dataset could be visualized in brat, we -->
    <!-- 	have only selected freely available subsets of those datasets -->
    <!-- 	that do not require a separate licence.) -->
</p>

<br style="clear:both"/>
<!-- ############################################################ -->

<h2 class="nomargin">Entity mention detection</h2>

<a href="img/examples/esp.train-doc-536-full.png">
    <img class="right" src="img/examples/esp.train-doc-536-small.png" alt="annotation example"/>
</a>

<h3>Example: CoNLL 2002 Shared Task: Language-Independent Named Entity Recognition</h3>


<p>
    The Conference on Computational Natural Language Learning
	(CoNLL) 2002 shared task on Language-Independent Named Entity
	Recognition provided two annotated corpora (Spanish and
	Dutch) annotated with entities of four types (person,
	organization, location and miscellaneous).
</p>

<p>
    A conversion script from the CoNLL 2002 shared task format into
    the brat standoff format and a sample of the corpus annotations
    are distributed with brat.
</p>

<p>
    The full shared task data are freely available from the shared
    task website.
</p>

<div class="inforow">
    <span class="infolabel">Examples in brat:</span>
<!--	<span class="infovalue"><a target="brat-example" href="{{ site.dist.demo_url }}/latest/#/not-editable/CoNLL-ST_2002_train/">see it live on brat</a> (read-only data)</span>-->
</div>

<div class="inforow">
    <span class="infolabel">Project website:</span>
    <span class="infovalue"><a href="http://www.cnts.ua.ac.be/conll2002/ner/">CoNLL 2002 shared task website</a></span>
</div>

<div class="inforow">
    <span class="infolabel">Publications:</span>
    <span class="infovalue"><a href="http://www.cnts.ua.ac.be/conll2002/pdf/15558tjo.pdf">Tjong Kim Sang, 2002</a> [PDF]</span>
</div>
      
<br style="clear:both"/>

<!-- ############################################################ -->

<h2 class="nomargin">Event extraction</h2>

<a href="img/examples/PMID-20300060-full.png">
    <img class="right" src="img/examples/PMID-20300060-small.png" alt="annotation example"/>
</a>

<h3>Example: BioNLP shared task: biomedical event extraction</h3>

<p>
    BioNLP shared task events held in 2009 and 2011 and the ongoing
    <a href="http://2013.bionlp-st.org">BioNLP Shared Task 2013</a>
    a have included several event extraction tasks.
</p>

<p>
    The brat standoff format is compatible with the data
    distribution format of the BioNLP shared task, and samples of
	the annotations of the various corpora provided for the task
	are distributed with brat.
</p>

<p>
    The full shared task data are freely available from the shared
    task website.
</p>

<div class="inforow">
	<span class="infolabel">Examples in brat:</span>
<!--    <span class="infovalue"><a target="brat-example" href="{{ site.dist.demo_url }}/latest/#/not-editable/BioNLP-ST_2011_ID_devel/">see it live on brat</a> (read-only data)</span>-->
</div>

<div class="inforow">
    <span class="infolabel">Project website:</span>
    <span class="infovalue"><a href="http://2011.bionlp-st.org">BioNLP shared task 2011 website</a></span>
</div>

<div class="inforow">
    <span class="infolabel">Publications:</span>
    <span class="infovalue"><a href="http://aclweb.org/anthology-new/W/W09/W09-1401.pdf">Kim et al. 2009</a>, <a href="http://aclweb.org/anthology-new/W/W11/W11-1801.pdf">2011</a> [PDFs]</span>
</div>

<br style="clear:both"/>

<!-- ############################################################ -->

<h2 class="nomargin">Coreference resolution</h2>

<a href="img/examples/PMID-1492121-full.png">
    <img class="right" src="img/examples/PMID-1492121-small.png" alt="annotation example"/>
</a>

<h3>Example: CO supporting task: coreference in scientific publications</h3>

<p>
    The BioNLP shared task 2011 included a supporting task
    on coreference in scientific publications.
</p>

<p>
    The brat standoff format is compatible with the
	representation used in the coreference task, and samples of
	the annotations of the corpus provided for the task are
	distributed with brat.
</p>

<p>
    The full shared task data are freely available from the shared
	task website.
</p>

<div class="inforow">
	<span class="infolabel">Examples in brat:</span>
<!--	<span class="infovalue"><a target="brat-example" href="{{ site.dist.demo_url }}/latest/#/not-editable/BioNLP-ST_2011_coreference_development_data/">see it live on brat</a> (read-only data)</span>-->
</div>

<div class="inforow">
	<span class="infolabel">Project website:</span>
	<span class="infovalue"><a href="http://2011.bionlp-st.org/home/protein-gene-coreference-task">BioNLP ST 2011 CO task home page</a></span>
</div>

<div class="inforow">
	<span class="infolabel">Publications:</span>
	<span class="infovalue"><a href="http://aclweb.org/anthology-new/W/W11/W11-1811.pdf">Nguyen et al. 2011</a> [PDF]</span>
</div>

<br style="clear:both"/>

<!-- ############################################################ -->

<h2 class="nomargin">Normalization</h2>

<a href="img/examples/CRAFT-16507151-full.png">
    <img class="right" src="img/examples/CRAFT-16507151-small.png" alt="visualization example"/>
</a>

<h3>Example: the Colorado Richly Annotated Full Text Corpus (CRAFT)</h3>
      
<p>
    The
	<a href="http://bionlp-corpora.sourceforge.net/CRAFT/index.shtml">Colorado
	Richly Annotated Full Text Corpus</a> (CRAFT) is an
	annotated corpus of full text biomedical journal
	publications. Mentions of concepts from seven biomedical
	ontologies/ terminologies, including 
	<a href="http://www.ebi.ac.uk/chebi/">ChEBI</a>, 
	<a href="http://www.geneontology.org/">GO</a>, and the
	<a href="http://www.ncbi.nlm.nih.gov/taxonomy">NCBI
	Taxonomy</a> are annotated and normalized to the appropriate
	ontology identifier.
</p>

<p>
    The corpus creators have converted the annotation into
	the brat format and provide an online service for
	<a href="http://compbio.ucdenver.edu/Craft/index.xhtml#/">browsing
	the CRAFT concept annotations</a> in brat.
</p>
      
<p>
    The full corpus is freely available from the project
    homepage.
</p>

<div class="inforow">
	<span class="infolabel">Examples in brat:</span>
<!--	<span class="infovalue"><a target="brat-example" href="http://compbio.ucdenver.edu/Craft/index.xhtml">see it live on brat</a> (read-only data)</span>-->
</div>

<div class="inforow">
	<span class="infolabel">Project website:</span>
	<span class="infovalue"><a href="http://bionlp-corpora.sourceforge.net/CRAFT/index.shtml">CRAFT home page</a></span>
</div>

<div class="inforow">
	<span class="infolabel">Publications:</span>
	<span class="infovalue"><a href="http://www.biomedcentral.com/1471-2105/13/161/">Bada et al. 2012</a>,
	<a href="http://www.biomedcentral.com/1471-2105/13/207">Verspoor et al. 2012</a></span>
</div>

<br style="clear:both"/>

<!-- ############################################################ -->

<a href="img/examples/EVEX-PMID-9144171-full.png">
	<img class="right" src="img/examples/EVEX-PMID-9144171-small.png" alt="visualization example"/>
</a>

<h3>Example: EVEX: large-scale event extraction resource with multi-level gene normalization</h3>
      
<p>
    The
    <a href="http://evexdb.org/">EVEX database</a> builds on an
	automatically annotated corpus covering 22 million PubMed
	abstracts and 460 thousand open access full-text articles from
	PubMed Central, created using tools such as the
	<a href="http://jbjorne.github.com/TEES/">Turku Event
	  Extraction System</a> and the gene normalization system
	<a href="http://ikmbio.csie.ncku.edu.tw/GN/">GenNorm</a>.
	Each gene and protein mention in the data is normalized to
	several databases at varying levels of granularity, ranging
	from canonical symbols to broad gene families and unique
	Entrez Gene IDs.
</p>

<p>
    The corpus creators have converted the annotation into
    the brat format. A (very) small sample is linked below.
</p>

<p>
    The full corpus is freely available from 
    <a href="http://bionlp.utu.fi/pubmedevents.html">bionlp.utu.fi/pubmedevents.html</a>.
</p>

<!--
<div class="inforow">
	<span class="infolabel">Examples in brat:</span>
	<span class="infovalue"><a target="brat-example" href="http://weaver.nlplab.org/~brat/demo/latest/#/not-editable/EVEX/">see it live on brat</a> (read-only data)</span>
</div>
-->

<div class="inforow">
	<span class="infolabel">Project website:</span>
	<span class="infovalue"><a href="http://evexdb.org/">EVEX home page</a></span>
</div>

<div class="inforow">
	<span class="infolabel">Publications:</span>
	<span class="infovalue"><a href="http://www.aclweb.org/anthology-new/W/W11/W11-0204.pdf">Van Landeghem et al. 2011</a>
</div>

<br style="clear:both"/>

<!-- ############################################################ -->

<h2 class="nomargin">Chunking</h2>

<p>
    Chunking is the task of dividing text into non-overlapping
    segments that are typically further assigned labels such as
	<span class="tt">NP</span> (Noun Phrase).
</p>

<a href="img/examples/train.txt-doc-109-full.png">
    <img class="right" src="img/examples/train.txt-doc-109-small.png" alt="annotation example"/>
</a>

<h3>Example: CoNLL 2000 Shared Task: Chunking</h3>
      
<p>
    The Conference on Computational Natural Language
	Learning 2000 (CoNLL 2000) shared task on chunking provides
	freely available training and test data.
</p>

<p>
    A conversion script from the CoNLL 2000 shared task format
	into the brat standoff format and a sample of the corpus
	annotations are distributed with brat.
</p>

<p>
    The full shared task data are freely available from the shared
    task website.
</p>

<div class="inforow">
	<span class="infolabel">Examples in brat:</span>
<!--	<span class="infovalue"><a target="brat-example" href="{{ site.dist.demo_url }}/latest/#/not-editable/CoNLL-00-Chunking">see it live on brat</a> (read-only data)</span>-->
</div>

<div class="inforow">
	<span class="infolabel">Project website:</span>
	<span class="infovalue"><a href="http://www.clips.ua.ac.be/conll2000/chunking/">CoNLL 2000 shared task website</a></span>
</div>

<div class="inforow">
    <span class="infolabel">Publications:</span>
    <span class="infovalue"><a href="http://www.clips.ua.ac.be/conll2000/pdf/12732tjo.pdf">Tjong Kim Sang and Buchholz (2000)</a> [PDF]</span>
</div>

<br style="clear:both"/>

<!-- ############################################################ -->

<h2 id="dependency_syntax" class="nomargin">Dependency syntax</h2>

<p>
    Dependency parsing (syntactic analysis) is the task of
    assigning binary relations between words to mark their
    head-dependent relations.
</p>

<a href="img/examples/swedish_talbanken05_train.conll-doc-880-full.png">
    <img class="right" src="img/examples/swedish_talbanken05_train.conll-doc-880-small.png" alt="annotation example"/>
</a>

<h3>Example: CoNLL-X Shared Task: Multi-lingual Dependency Parsing</h3>
      
<p>
    The Tenth Conference on Computational Natural Language
	Learning (CoNLL-X) shared task on Multi-lingual Dependency
	Parsing provided annotated corpora for 13 languages, four of
	which are freely availabe (for Danish, Dutch, Portuguese and
	Swedish).
</p>

<p>
    A conversion script from the CoNLL-X shared task format into
	the brat standoff format and a sample of the corpus
	annotations for these four languages are distributed with
	brat.
</p>

<p>
    The full shared task data are freely available from the shared
	task website.
</p>

<div class="inforow">
	<span class="infolabel">Examples in brat:</span>
<!--	<span class="infovalue"><a target="brat-example" href="{{ site.dist.demo_url }}/latest/#/not-editable/CoNLL-X-Swedish">see it live on brat</a> (read-only data)</span>-->
</div>

<div class="inforow">
	<span class="infolabel">Project website:</span>
	<span class="infovalue"><a href="http://ilk.uvt.nl/conll/">CoNLL-X shared task website</a></span>
</div>

<div class="inforow">
	<span class="infolabel">Publications:</span>
	<span class="infovalue"><a href="http://aclweb.org/anthology-new/W/W06/W06-2920.pdf">Buchholz and Marsi (2006)</a> [PDF]</span>
</div>

<a href="img/examples/TDT-w078-full.png">
    <img class="right" src="img/examples/TDT-w078-small.png" alt="annotation example"/>
</a>

<h3>Example: Turku Dependency Treebank (TDT)</h3>

<p>
    TDT is a Finnish treebank manually annotated using the Stanford
    Dependencies (SD) scheme. In addition to the base syntactic trees, TDT
    also includes additional relations, such as coordination expansion.
    The underlying corpus comprises a number of different genres,
    including among others legalese, encyclopedic text, fiction, and news.

    A conversion script from TDT's native XML format into the brat
    standoff format is available upon request and a sample of the treebank
    is distributed with brat.
</p>

<div class="inforow">
	<span class="infolabel">Examples in brat:</span>
<!--	<span class="infovalue"><a target="brat-example" href="{{ site.dist.demo_url }}/latest/#/not-editable/Turku_Dependency_Treebank">see it live on brat</a> (read-only data)</span>-->
</div>

<div class="inforow">
	<span class="infolabel">Project website:</span>
	<span class="infovalue"><a href="http://bionlp.utu.fi/fintreebank.html">TDT website</a></span>
</div>

<div class="inforow">
	<span class="infolabel">Publications:</span>
	<span class="infovalue"><a href="http://hdl.handle.net/10062/15936">Haverinen et al. (2010)</a></span>
</div>

<br style="clear:both"/>

<!-- ############################################################ -->

<h2 class="nomargin">Meta-knowledge</h2>

<p>
   Meta-knowledge annotation is the task of identifying how factual
   statements should be interpreted, according to their textual context,
   Examples include whether a statement describing a fact, a hypothesis,
   an experimental result or an analysis of results, how confident the
   author is regarding the validity of her analyses, etc.
</p>

<a href="img/examples/nactem-metaknowledge-1545132-full.png">
    <img class="right" src="img/examples/nactem-metaknowledge-1545132-small.png" alt="annotation example"/>
</a>

<h3>Example: Meta-knowledge enrichment of the GENIA Event Corpus</h3>
      
<p>
</p>

<p>
    The complete GENIA event copus, consisting of 1,000 abstracts annotated with 36,858 events, has been manually enriched with meta-knowledge annotation. This will faciltate the training of more sophisticated IE systems, which allow interpretative information to be specified as part of the search criteria. Such systems can assist in a number of important tasks, e.g., finding new experimental knowledge to facilitate database curation, enabling textual inference to detect entailments and contradictions, etc.
</p>

<p>
    Each event has been annotated with 5 different dimensions of meta-knowledge, i.e. knowledge type (KT), certainty level (CL), manner, polarity and source. The annotation consists of assigning an appopriate value for each dimension and identifying textual clues for each dimension, when they are present.
</p>

<p>
    The meta-knowledge enririched GENIA corpus is freely available from the project website.
</p>

<div class="inforow">
	<span class="infolabel">Examples in brat:</span>
<!--	<span class="infovalue"><a target="brat-example" href="http://www.nactem.ac.uk/brat/#/GENIA-Metaknowledge/">see it live on brat</a> (read-only data)</span>-->
</div>

<div class="inforow">
	<span class="infolabel">Project website:</span>
	<span class="infovalue"><a href="http://www.nactem.ac.uk/meta-knowledge/">NaCTeM Meta-knowledge Project</a></span>
</div>

<div class="inforow">
	<span class="infolabel">Publications:</span>
	<span class="infovalue"><a href="http://www.biomedcentral.com/1471-2105/12/393/abstract">Thompson et al. (2011)</a></span>
</div>

<br style="clear:both"/>

<!-- ############################################################ -->

<h1 id="corpus-examples-brat">corpora annotated using brat</h1>

<p>
    This section presents a selection of corpora annotated using
    brat.
</p>

<br style="clear:both"/>

<!-- ############################################################ -->

<h2 class="nomargin">Information Extraction</h2>

<p>
    Information Extraction is the task of capturing essential information
    contained in text and convert it into a formal representation suitable for
    automatic processing.
</p>

<a href="img/examples/PMID-7806663-full.png">
    <img class="right" src="img/examples/PMID-7806663-small.png" alt="annotation example"/>
</a>

<h3 class="nomargin">Anatomical Entity Mention (AnEM) corpus</h3>

<p>
    brat was used to create AnEM, a corpus of 500 documents drawn
	from PubMed abstracts and PMC full text extracts, annotated
	for 11 classes of anatomical entities. The corpus is intended
	to serve as a reference for training and evaluating methods
	for anatomical entity mention detection in life science
	publications.
</p>

<div class="inforow">
    <span class="infolabel">Examples in brat:</span>
<!--	<span class="infovalue"><a target="brat-example" href="{{ site.dist.demo_url }}/latest/#/not-editable/AnEM-1.0.4/">see it live on brat</a> (read-only data)</span>-->
</div>

<div class="inforow">
	<span class="infolabel">Project website:</span>
    <span class="infovalue"><a href="http://www.nactem.ac.uk/anatomy/">Nactem anatomy page</a></span>
</div>

<div class="inforow">
	<span class="infolabel">Publications:</span>
	<span class="infovalue"><a href="http://www.nactem.ac.uk/anatomy/docs/ohta2012opendomain.pdf">Ohta et al., 2012</a> [PDF]</span>
</div>

<br style="clear:both"/>

<!-- ############################################################ -->

<a href="img/examples/cellfinder_corpus2-full.png">
    <img class="right" src="img/examples/cellfinder_corpus2-small.png" alt="annotation example"/>
</a>

<h3 class="nomargin">CellFinder corpus</h3>

<p>
    brat was used to annotate the CellFinder corpus of 10 full
	papers annotated with six biomedical entities: genes/proteins,
	cell components, cell types, cell lines, anatomical parts
	(tissues/organs) and organisms. The corpus was developed in
	the scope of the CellFinder project which aims to establish a
	central stem cell data repository.
</p>

<div class="inforow">
	<span class="infolabel">Examples in stav:</span>
<!--	<span class="infovalue"><a target="brat-example" href="http://corpora.informatik.hu-berlin.de/index.xhtml#/cellfinder/version1_sections/">see it live</a> (read-only data)</span>-->
</div>

<div class="inforow">
	<span class="infolabel">Project website:</span>
	<span class="infovalue"><a href="http://www.informatik.hu-berlin.de/forschung/gebiete/wbi/resources/cellfinder/">CellFinder project resource page</a></span>
</div>

<div class="inforow">
	<span class="infolabel">Publications:</span>
	<span class="infovalue"><a href="http://www.informatik.hu-berlin.de/forschung/gebiete/wbi/research/publications/2012/lrec2012_corpus.pdf">Neves et al., 2012</a> [PDF]</span>
</div>

<br style="clear:both"/>

<!-- ############################################################ -->

<a href="img/examples/MLEE-PMID-15975645-full.png">
    <img class="right" src="img/examples/MLEE-PMID-15975645-small.png" alt="annotation example"/>
</a>

<h3 class="nomargin">Multi-Level Event Extraction (MLEE) corpus</h3>

<p>
    brat was used to create the Multi-Level Event Extraction
	(MLEE) corpus of 260 PubMed abstracts annotated with 16
	physical biomedical domain entity types ranging from molecular
	entities to cells, tissues, and organs and 29 event types
	involving these entities, defined primarily with reference to
	the <a href="http://www.geneontology.org/">Gene Ontology</a>.
</p>

<p>The corpus is currently being applied and extended in the
    <a href="">BioNLP Shared Task 2013 Cancer Genetics task</a>.
</p>

<div class="inforow">
    <span class="infolabel">Examples in brat:</span>
<!--    <span class="infovalue"><a target="brat-example" href="http://www.nactem.ac.uk/eccb2012/index.xhtml#">see it live</a> (read-only data)</span>-->
</div>

<div class="inforow">
    <span class="infolabel">Project website:</span>
    <span class="infovalue"><a href="http://nactem.ac.uk/MLEE/">Multi-Level Event Extraction (MLEE) project homepage</a></span>
</div>

<div class="inforow">
    <span class="infolabel">Publications:</span>
    <span class="infovalue"><a href="http://bioinformatics.oxfordjournals.org/content/28/18/i575.full">Pyysalo et al., 2012</a></span>
</div>

<br style="clear:both"/>

<!-- ############################################################ -->

<h2 class="nomargin">Metaphor annotation through bottom-up identification</h2>

<p>
    Bottom-up metaphor annotation is the task of identifying linguistic
    metaphors in the text and inferring the conceptual metaphors that underlie
    them.
</p>

<a href="img/examples/russian_metaphor-full.png">
    <img class="right" src="img/examples/russian_metaphor-small.png" alt="annotation example"/>
</a>

<h3 class="nomargin">Russian-language corpus of conceptual metaphor</h3>

<p>
    The manually annotated Russian-language corpus of conceptual metaphor is
    an ongoing project in the initial stage.
    The annotation is performed at the two levels.
    At the shallow level, metaphor-related words are annotated according to
    slightly modified rules of MIPVU, the linguistic procedure of metaphor
    identification. Each metaphor-related word is assigned to one of the 10
    classes.
    The word meanings that form the basis of metaphoricity are described for
    each metaphor-related word.
</p>

<p>
    The deep-level annotation consists in annotating metaphorical mappings
    between the conceptual domains of the Source and the Target, and inferring
    conceptual metaphors from them.
</p>

<p>
    This scheme is suitable for identifying and annotating conceptual
    metaphors that are partly manifested in the text, with only the Source
    being present explicitly, while the Target and the mapping are implicit.
</p>

<!--<div class="inforow">
    <span class="infolabel">Examples in brat:</span>
    <span class="infovalue"><a target="brat-example" href="XXX">see it live</a> (read-only data)</span>
</div>

<div class="inforow">
    <span class="infolabel">Project website:</span>
    <span class="infovalue"><a href="XXX">XXX</a></span>
</div>-->

<div class="inforow">
    <span class="infolabel">Publication:</span>
    <a href="http://www.aclweb.org/anthology/W13-0910">Badryzlova et al. (2014)</a>
    <!--<span class="infovalue"><a href="XXX">XXX</a></span>-->
</div>

<br style="clear:both"/>

<!-- ############################################################ -->

<h1 id="other-uses">other uses of brat</h1>

<p>
    brat can be used also for tasks other than manual annotation.
</p>

<br style="clear:both"/>

<!-- ############################################################ -->

<h2 class="nomargin">Visualisation</h2>

<p>
    brat has not just been used to create annotations but also to visualise
    annotation output from other tools for manual analysis and evaluation
    purposes.
    Instructions on how to embed brat visualisations in web pages are
    available <a href="embed.html">here</a>.
</p>

<a href="img/examples/Stanford-CoreNLP-full.png">
	<img class="right" src="img/examples/Stanford-CoreNLP-small.png" alt="visualization example"/>
</a>

<h3 class="nomargin">Natural language analysis tool visualization</h3>

<p>
    The <a href="http://nlp.stanford.edu/">Standford NLP
	group</a> is using brat to provide visualizations of the
	annotations created by the 
	<a href="http://nlp.stanford.edu:8080/corenlp/">live online
	demo of the Stanford CoreNLP</a> natural language analysis
	tools.
</p>

<p>
    The
    <a href="http://nlp.stanford.edu/software/corenlp.shtml">CoreNLP</a>
    tools perform part-of-speech tagging, named entity
    recognition, coreference resolution, and syntactic analysis.
	The demonstration provides separate visualizations of all of
    these annotation layers using
    <a href="embed.html">embedded brat</a>.
</p>

<div class="inforow">
	<span class="infolabel">Examples in brat:</span>
	<span class="infovalue"><a target="brat-example" href="http://nlp.stanford.edu:8080/corenlp/">see it live</a> (online demo)</span>
</div>

<div class="inforow">
	<span class="infolabel">Project website:</span>
	<span class="infovalue"><a href="http://nlp.stanford.edu/software/corenlp.shtml">Stanford CoreNLP home page</a></span>
</div>

<div class="inforow">
	<span class="infolabel">Publications:</span>
	<span class="infovalue">
	  <a href="http://nlp.stanford.edu/~manning/papers/tagging.pdf">Toutanova et al. 2003</a>,
	  <a href="http://nlp.stanford.edu/~manning/papers/gibbscrf3.pdf">Finkel et al. 2005</a>,
	  <a href="http://nlp.stanford.edu/~manning/papers/unlexicalized-parsing.pdf">Klein and Manning 2003</a>,
      <a href="http://nlp.stanford.edu/pubs/coreference-emnlp10.pdf">Raghunathan et al. 2010</a> [PDFs].
  </span>
</div>

<br style="clear:both"/>

<!-- ############################################################ -->

<h3 class="nomargin">Information Extraction system evaluation</h3>

<a href="img/examples/PMID-10860807-eval-full.png">
    <img class="right" src="img/examples/PMID-10860807-eval-small.png" alt="evaluation example"/>
</a>

<h3>Example: BioNLP shared task: biomedical event extraction</h3>

<p>
    BioNLP Shared Task 2011 and the ongoing
	<a href="http://2013.bionlp-st.org">BioNLP Shared Task 2013</a>
	are using brat to provide visualizations of the results of
	information extraction system predictions compared to
	gold standard annotations.
</p>

<p>
    The evaluation system evaluates differences between two sets
	of annotations and generates a visual comparison in brat
	format, using brat features to make differences visually
	obvious.
</p>

<!--       <div class="inforow"> -->
<!-- 	<span class="infolabel">Examples in brat:</span> -->
<!-- 	<span class="infovalue"><a target="brat-example" href="http://weaver.nlplab.org/~brat/demo/latest/#/not-editable/BioNLP-ST_2011_ID_devel/">see it live on brat</a> (read-only data)</span> -->
<!--       </div> -->
<!--       <div class="inforow"> -->
<!-- 	<span class="infolabel">Project website:</span> -->
<!-- 	<span class="infovalue"><a href="http://2011.bionlp-st.org">BioNLP shared task 2011 website</a></span> -->
<!--       </div> -->
<!--       <div class="inforow"> -->
<!-- 	<span class="infolabel">Publications:</span> -->
<!-- 	<span class="infovalue"><a href="http://aclweb.org/anthology-new/W/W09/W09-1401.pdf">Kim et al. 2009</a>, <a href="http://aclweb.org/anthology-new/W/W11/W11-1801.pdf">2011</a> [PDFs]</span> -->
<!--       </div> -->

<br style="clear:both"/>

<div>
    <h2 style="text-align:right"><a href="features.html">Next: features in overview</a></h2>
</div>
